charts/gcp-workload-identity-federation-webhook/templates/deployment.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "gcp-workload-identity-federation-webhook.fullname" . }}-controller-manager
```

Comment:
metadata.namespace field is not automatically set if using `helm template` command.

```
$ helm version
version.BuildInfo{Version:"v3.9.0", GitCommit:"7ceeda6c585217a19a1131663d8cd1f7d641b2a7", GitTreeState:"clean", GoVersion:"go1.17.5"}
$ helm template charts/gcp-workload-identity-federation-webhook | grep metadata -A 10 -B 10 | head -n 15
---
# Source: gcp-workload-identity-federation-webhook/templates/deployment.yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: release-name-gcp-wif-webhook-controller-manager
  labels:
    helm.sh/chart: gcp-workload-identity-federation-webhook-0.0.0
    app.kubernetes.io/name: gcp-wif-webhook
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/version: "0.0.0"
    app.kubernetes.io/managed-by: Helm
---
# Source: gcp-workload-identity-federation-webhook/templates/manager-rbac.yaml
apiVersion: rbac.authorization.k8s.io/v1
```

The below change is user-friendly for `helm template` users.

```suggestion
metadata:
  name: {{ include "gcp-workload-identity-federation-webhook.fullname" . }}-controller-manager
  namespace: {{ .Release.Namespace }}
```

Reply:
👍